app/model/GuardianWeeklyZones.scala

```diff
@@ -25,3 +26,20 @@ object GuardianWeeklyZones {
   val domesticZoneCountries = domesticZoneCountryGroups.flatMap(_.countries)
   val restOfWorldZoneCountries = allCountries.diff(domesticZoneCountries)
 }
+
+object PurchasableWeeklyProducts {
+
+  sealed trait PurchasableWeeklyProduct
```

Comment:
This allows us to test that the right product is returned before/after switchover for various regions, without worrying about all of the plans in the catalog